code/utils/scripts/get_pcs_script.py

```diff
@@ -27,8 +27,7 @@
 from Image_Visualizing import make_mask
 
 # List of subject directories.
-#sub_list = os.listdir(path_to_data)
-sub_list = ['sub002', 'sub003', 'sub014'] # Just 3 for now.
+sub_list = os.listdir(path_to_data)
```

Comment:
It was always meant to be sub_list = os.listdir(path_to_data), but when you guys were experimenting with things and just wanted to run stuff on a few subjects (for time costs), I wrote in "sub_list = ['sub002', 'sub003', 'sub014']" so you use the script to do PCA on just those subjects. I changed it back-- if anybody wants to do PCA on a subset of subjects, they can just alter this script or write a modified version to suit their purposes.